### PR TITLE
Added Chetan-07 to CoP

### LINKED ISSUE #539: Please add me as a member of the redhat-cop organization
I would like to become a general contributor to the Red Hat Communities of Practice GitHub Organization. 

To be a member of the Red Hat CoP GitHub org, you are required to be a Red Hat employee.
Non-employees are invited to be outside-collaborators (https://github.com/orgs/redhat-cop/outside-collaborators).
For this reason, please complete the below form as well:
- https://red.ht/github-redhat-cop-username

---

Review request:

cc: @redhat-cop/container-cop-core

### PR DESCRIPTION
**Pull request by sabre1041**
Added Chetan-07 to CoP to resolve #539

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -54,6 +54,7 @@ orgs:
       - caseyoc
       - cescoffier
       - chadmf
+      - Chetan-07
```
Comment (pabrahamsson) [inline config.yaml:57]:
```suggestion
      - chetan-07
```

**Review by pabrahamsson — CHANGES_REQUESTED** [review]
As per #514 we've moved to using lowercase usernames for consistency. Would you mind updating @sabre1041?

**Comment by sabre1041** [comment]
@pabrahamsson updated to lowercase

**Review by pabrahamsson — APPROVED** [review]
LGTM

**Pull request merged**